Modules/TRD/RawData.json

```diff
@@ -0,0 +1,66 @@
+{
+  "qc": {
+    "config": {
+      "database": {
+        "implementation": "CCDB",
+        "host": "ccdb-test.cern.ch:8080",
+        "username": "not_applicable",
+        "password": "not_applicable",
+        "name": "not_applicable"
+      },
+      "Activity": {
+        "number": "504419",
+        "type": "2",
+        "periodName": "",           "": "Period name - e.g. LHC22c, LHC22c1b_test",
+        "passName": "",             "": "Pass type - e.g. spass, cpass1",
+        "provenance": "qc",         "": "Provenance - qc or qc_mc depending whether it is normal data or monte carlo data"
+      },
+      "monitoring": {
+        "url": "infologger:///debug?qc"
+      },
+      "consul": {
+        "url": "http://consul-test.cern.ch:8500"
```

Comment:
Please remove this default consul instance. It is not used anymore. Just leave it blank.

Reply:
cool, done. The actual working json file is in my home directory on the epn. Should the current json files reside in the git repo?

Reply:
the json file in git is usually the one you use to test locally your code.